stats/analytics.py

```diff
@@ -149,6 +151,45 @@ def get_codelist_mapping(major_version):
         )
 
 
+def build_org_prefix_list():
+    """Build lists of valid organisation identifier prefixes"""
+    out = {}
+    for major_version in ('1', '2'):
+        out[major_version] = defaultdict(list)
+        for prefix in CODELISTS[major_version]['OrganisationRegistrationAgency']:
+            out[major_version][len(prefix)].append(prefix)
+    return out
+
+
+org_prefix_list = build_org_prefix_list()
+
+
+def build_channel_code_list():
+    """Build lists of CRS Channel Codes"""
+    out = {}
+    for major_version in ('1', '2'):
+        out[major_version] = defaultdict(list)
+        for code in CODELISTS[major_version]['CRSChannelCode']:
+            out[major_version][code[:2]].append(code)
```

Comment:
Same sort of question [as above](https://github.com/codeforIATI/IATI-Stats/pull/34/files#r1010363895) – why group by the first bit of the channel code?